config/config.go

```diff
@@ -74,6 +75,19 @@ func AppRoot() string {
 	}
 }
 
+// ResolvePath returns an absolute path derived from AppRoot and the relative path.
+// If the input parameter is already an absolute path it will be returned immediately.
+func ResolvePath(relative string) (string, error) {
```

Comment:
this is used by fx consumers. there's a diff out that uses it in the uber private repo. i'll also be using it in other projects.